server/http.go

```diff
@@ -241,20 +234,17 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 	post.Message = "Meeting has ended."
 	post.Props["meeting_status"] = zoom.WebhookStatusEnded
 
-	_, appErr = p.API.UpdatePost(post)
-	if appErr != nil {
+	if _, appErr := p.API.UpdatePost(post); appErr != nil {
 		http.Error(w, appErr.Error(), appErr.StatusCode)
 		return
 	}
 
-	appErr = p.API.KVDelete(key)
-	if appErr != nil {
+	if appErr := p.API.KVDelete(key); appErr != nil {
 		p.API.LogWarn("failed to delete db entry", "error", appErr.Error())
 		return
```

Comment:
Nit: It's fine to continue on warnings
```suggestion
		p.API.LogWarn("failed to delete db entry", "error", appErr.Error())
```